GPyOpt/plotting/plots_bo.py

```diff
@@ -2,149 +2,183 @@
 # Licensed under the BSD 3-clause license (see LICENSE.txt)
 
 import numpy as np
-from pylab import grid
 import matplotlib.pyplot as plt
-from pylab import savefig
-import pylab
 
 
-def plot_acquisition(bounds,input_dim,model,Xdata,Ydata,acquisition_function,suggested_sample, filename = None):
+def _normalize_acqu(acqu, m, pct95):
```

Comment:
Again, let's refraining from unnecessary shortenings. Also this method could use some docstring to explain what it's for.